docker/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM debian:bullseye-slim
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV LANG=C.UTF-8
+ENV LC_ALL=C.UTF-8
+
+RUN echo '\
+Acquire::http::No-Cache true;\
+APT::Get::Assume-Yes "true";\
+APT::Install-Recommends "false";\
+APT::Install-Suggests "false";\
+' > /etc/apt/apt.conf.d/99custom
+
+RUN apt-get update && apt-get upgrade
+RUN apt-get install curl
+RUN apt-get install ca-certificates
+
+ENV PWD="/usr/app"
+WORKDIR $PWD
+ENV version="0.1.5"
```

Comment:
this will quickly get out-of-date. could you add it to goreleaser manifest? https://goreleaser.com/customization/docker/

this also could be packaged to gh container registry: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry

Reply:
Of course, as soon as possible I'll take a look at it and give you a feedback.

Reply:
It's using GitHub releases to keep maintaining burden lower

Reply:
Done, I'll wait for you to config ghcr to make further adjustments